include/alpaka/workdiv/WorkDivHelpers.hpp

```diff
@@ -235,7 +235,7 @@ namespace alpaka
                 // For equal block thread extent this is easily the nth root of blockThreadCountMax.
                 if(gridBlockExtentSubDivRestrictions == GridBlockExtentSubDivRestrictions::EqualExtent)
                 {
-                    double const fNthRoot(std::pow(blockThreadCountMax, 1.0/static_cast<double>(TDim::value)));
+                    double const fNthRoot(std::pow(static_cast<double>(blockThreadCountMax), 1.0/static_cast<double>(TDim::value)));
```

Comment:
Is this change required for `apt-get` to work??

Reply:
This is not directly related to apt-getting clang but by doing so, we now get the correct std lib installed automatically instead of always using the one we defined with `ALPAKA_CI_CLANG_LIBSTDCPP_VERSION`, which was an old one. The newer std-lib seems to provide multiple equally matching overloads of `std::pow` and compilation failed.